consensus/consortium/v2/consortium.go

```diff
@@ -741,6 +745,8 @@ func (c *Consortium) getCheckpointValidatorsFromContract(
 					BlsPublicKey: blsPublicKey,
 					Weight:       weights[i],
 				})
+			} else {
+				log.Error("failed to get bls public key", "err", err, "candidate", candidate, "number", header.Number)
```

Comment:
Should be log.Info
Should be "Failed"

Reply:
Why log.Warn? There is case like in testnet, while migrating, some validators does not input their BLS key, so this leads to error in this case but there is not thing wrong with the node or contract side so I think log.Info is enough.